### PR TITLE
Update practical-tb.Rmd

### PR DESCRIPTION
**Pull request by juanvesga**
Now fixed tiny changes

### CODE REVIEW

File: content/post/practical-tb.Rmd [hunk 1]
```diff
@@ -446,13 +446,13 @@ out0  <- get_intervention(xstart, params, NA, times, NA, NA, TB.Basic,
                           "Initial", NA) 
 
 # plot
-dot <- data.frame(Data = "Bangladesh", Years = 2017, incidence = Inc.country)
```
Comment (zkamvar) [inline content/post/practical-tb.Rmd]: Hi @juanvesga,

This produces an error. Was this supposed to change to `country.name`?

**Comment by juanvesga** [comment]
Yes! My mistake

On 21 Jun 2019, at 08:47, Zhian N. Kamvar <notifications@github.com<mailto:notifications@github.com>> wrote:


@zkamvar commented on this pull request.

________________________________

In content/post/practical-tb.Rmd<https://github.com/reconhub/learn/pull/82#discussion_r296242684>:

> @@ -446,13 +446,13 @@ out0  <- get_intervention(xstart, params, NA, times, NA, NA, TB.Basic,
                           "Initial", NA)

 # plot
-dot <- data.frame(Data = "Bangladesh", Years = 2017, incidence = Inc.country)


Hi @juanvesga<https://github.com/juanvesga>,

This produces an error. Was this supposed to change to country.name?

**Comment by juanvesga** [comment]
Should I edit it again? Or can you change it?

Thanks,

Juan

On 21 Jun 2019, at 08:47, Zhian N. Kamvar <notifications@github.com<mailto:notifications@github.com>> wrote:


@zkamvar commented on this pull request.

________________________________

In content/post/practical-tb.Rmd<https://github.com/reconhub/learn/pull/82#discussion_r296242684>:

> @@ -446,13 +446,13 @@ out0  <- get_intervention(xstart, params, NA, times, NA, NA, TB.Basic,
                           "Initial", NA)

 # plot
-dot <- data.frame(Data = "Bangladesh", Years = 2017, incidence = Inc.country)


Hi @juanvesga<https://github.com/juanvesga>,

This produces an error. Was this supposed to change to country.name?

**Comment by zkamvar** [comment]
I can change it

**Comment by juanvesga** [comment]
Thanks!

On 21 Jun 2019, at 08:56, Zhian N. Kamvar <notifications@github.com<mailto:notifications@github.com>> wrote:


I can change it

**Pull request merged**